arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,332 @@
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                },
+                "supportedServiceLevelObjectives": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ServiceObjectiveCapability"
+                    },
+                    "description": "The list of supported Service Objectives for the edition."
+                }
+            },
+            "description": "Represents the server edition capabilities."
+        },
+        "ServerVersionCapability": {
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The server version name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status of the server version.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                },
+                "supportedEditions": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/EditionCapability"
+                    },
+                    "description": "The list of supported server editions."
+                }
+            },
+            "description": "Represents the server capabilities."
+        },
+        "LocationCapabilities": {
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The region name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status for the region with respect to Azure SQL.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                },
+                "supportedServerVersions": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ServerVersionCapability"
+                    },
+                    "description": "The list of supported server versions."
+                }
+            },
+            "description": "Represents the capabilities for a region."
+        },
+        "Resource": {
```

Comment:
oh things have changed (to the better) since then :-) as per the Swagger standard, you can now give references such as `./sql.core.json#/definitions/Resource`